packages/@okta/vuepress-site/docs/concepts/event-hooks/index.md

```diff
@@ -15,55 +15,55 @@ To handle event hook calls from Okta, you need to implement a web service with a
+Before the introduction of event hooks, polling the [System Log API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SystemLog/) was the only method your external software systems could use to detect the occurrence of specific events in your Okta org. Event hooks provide an Okta-initiated push notification.
 
 You can have a maximum of 25 active and verified event hooks set up in your org at any time. Each event hook can be configured to deliver multiple event types.
 
-> **Note:** To deliver event information, event hooks use the data structure associated with the [System Log API](/docs/reference/api/system-log/).
+> **Note:** To deliver event information, event hooks use the data structure associated with the [System Log API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SystemLog/).
 
 ## Which events are eligible?
 
 During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures.
 
 To see the list of event types currently eligible for use in event hooks, use the [Event types catalog](/docs/reference/api/event-types/#catalog) and search with the parameter `event-hook-eligible`.
 
-For general information on how Okta encapsulates events, see the [System Log API](/docs/reference/api/system-log/) documentation.
+For general information on how Okta encapsulates events, see the [System Log API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SystemLog/) documentation.
 
 Event types include user lifecycle changes, the completion by a user of a specific stage in an Okta process flow, and changes in Okta objects. You can configure an event hook, for example, to deliver notifications of user deactivation events. You can use hooks to trigger processes that you need to execute internally every time a user is deactivated. For example, updating a record in an HR system, creating a ticket in a support system, or generating an email message.
 
 <ApiLifecycle access="ea" />
 
-You can reduce the number of event hook calls by defining filters on specific instances of the subscribed event type. For example, if you want an event hook call triggered by user sign-in events for a specific group of users, you can filter on that group, rather than having an event hook call for every user sign in. See [Create an event hook filter](#create-an-event-hook-filter).
+You can reduce the number of event hook calls by defining filters on specific instances of the subscribed event type. For example, if you want an event hook call triggered by user sign-in events for a specific group of users, you can filter on that group, rather than having an event hook call for every user sign-in. See [Create an event hook filter](#create-an-event-hook-filter).
```

Comment:
```suggestion
You can reduce the number of event hook calls by defining filters on specific instances of the subscribed event type. For example, if you want an event hook call triggered by user sign-in events for a specific group of users, you can filter on that group. Do this rather than have an event hook call for every user sign-in flow. See [Create an event hook filter](#create-an-event-hook-filter).
```